bin/js-test

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+function cleanup {
+  cd ../..
+  docker compose -f docker-compose.test.yml down --rmi local --volumes
+}
+
+trap cleanup EXIT
+
+docker compose -f docker-compose.test.yml up --build -d
+
+cd clients/js 
+yarn
+yarn test:run
```

Comment:
wondering if this should be merged with `integration-test` to save on setup and tear down time

Reply:
@levand what do you think here?